lib/tranzito_utils/helpers/helpers.rb

```diff
@@ -0,0 +1,79 @@
+# frozen_string_literal: true
+
+module TranzitoUtils
+  module Helpers
+    include TranzitoUtils::SortableHelper
+    include TranzitoUtils::GraphingHelper
+
+    def controller_namespace
```

Comment:
`controller_namespace` needs to be defined in the controller because of something in another project ([bike_index](https://github.com/bikeindex/bike_index), which will eventually get `tranzito_utils`).

I know it's ugly, because it doesn't fit with `SetPeriod` exactly - but I think it makes sense to move `controller_namespace` into SetPeriod, so that it is included.